### PR TITLE
ios relative url

### PR DESCRIPTION
**Pull request by nesalang**
- Relative url implemented
- fixed empty url being tried
- added unit test for RU

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRenderer.mm [hunk 1]
```diff
@@ -97,14 +102,17 @@ + (UIView *)renderWithAdaptiveCards:(std::shared_ptr<AdaptiveCard> const &)adapt
         [rootView waitForAsyncTasksToFinish];
 
         UIView *leadingBlankSpace = nil, *trailingBlankSpace = nil;
-        if( adaptiveCard->GetVerticalContentAlignment() == VerticalContentAlignment::Center || adaptiveCard->GetVerticalContentAlignment() == VerticalContentAlignment::Bottom ){
+        if(adaptiveCard->GetVerticalContentAlignment() == VerticalContentAlignment::Center || 
+	       adaptiveCard->GetVerticalContentAlignment() == VerticalContentAlignment::Bottom){
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRenderer.mm]: >	 [](start = 0, length = 1)

No tabs please.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm [hunk 2]
```diff
@@ -370,8 +370,13 @@ - (void)loadImage:(std::string const &)urlStr
     }
     
     NSString *nSUrlStr = [NSString stringWithCString:urlStr.c_str()
-                                          encoding:[NSString defaultCStringEncoding]];
+                                            encoding:[NSString defaultCStringEncoding]];
     NSURL *url = [NSURL URLWithString:nSUrlStr];
+    // if url is nil, try again with adding base url from host config
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm]: >if url is nil, try again with adding base url from host config [](start = 7, length = 62)

This comment is wrong: you're not checking whether the URL is nil, but whether it's relative.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRenderer.mm (hunk 1 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRenderer.mm]: thanks!, my difftools added tabs.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm (hunk 2 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm]: thanks

**Pull request merged**